qcodes/loops.py

```diff
@@ -831,6 +831,10 @@ def _run_loop(self, first_delay=0, action_indices=(),
                 tprint('loop %s: %d/%d (%.1f [s])' % (
                     self.sweep_values.name, i, imax, time.time() - t0),
                     dt=self.progress_interval, tag='outerloop')
+                  if i:
```

Comment:
@kNalj You should make sure that your indention levels matches the fixed ones. In this case I think the  line should match the tprint above